src/message/MessageContainer.js

```diff
@@ -64,7 +64,7 @@ export default class MessageContainer extends React.PureComponent {
       pushRoute,
     } = this.props;
     const MessageComponent = isBrief ? MessageBrief : MessageFull;
-    const childrenNodes = htmlToDomTree(message.match_content || message.content);
+    const childrenNodes = htmlToDomTree((message.match_content || message.content).replace(/\r\n/g, '<br />').replace(/[\r\n]/g, '<br />'));
```

Comment:
Means we only want to this for content inside `pre` tag. 
@borisyankov can you please me give one or two cases where we are ignoring them, because I can't find any major change with this (where am I missing?).
@timabbott can we do this on server side (while sending message content to mobile client)?